data/json/items/melee.json

```diff
@@ -100,6 +100,25 @@
     "cutting": 8,
     "qualities": [ [ "COOK", 1 ], [ "HAMMER", 1 ] ]
   },
+ {
+    "id": "spear_stone",
+    "//": "Taken from More Surviver Tools",
+    "type": "GENERIC",
+    "name": "stone spear",
+    "description": "A stout wooden pole with a sharp stone spearhead.",
+    "weight": 1098,
+    "volume": 5,
+    "price": 1300,
+    "to_hit": 1,
+    "bashing": 5,
```

Comment:
This needs a breakdown of damage compared to wood, copper and steel spears.
It should be between wood and copper in overall damage output.
It's not as obvious as it might seem since this is currently significantly lighter than the copper spear, that might make it faster and therefore deal more damage than the copper spear.
It should also have the NONCONDUCTIVE flag.